requirements-git.txt

```
@@ -4,3 +4,4 @@ git+https://github.com/dolfin-adjoint/pyadjoint.git#egg=pyadjoint-ad
 git+https://github.com/firedrakeproject/loopy.git@main#egg=loopy
 git+https://github.com/firedrakeproject/pytest-mpi.git@main#egg=pytest-mpi
 git+https://github.com/firedrakeproject/petsc.git@firedrake#egg=petsc
+git+https://github.com/firedrakeproject/libsupermesh.git@connorjward/pip-install2
```

```suggestion
git+https://github.com/firedrakeproject/libsupermesh.git#egg=libsupermesh
```
unsure about this egg business

---

It just corresponds to the name of the underlying package, which may be different to the repo name. This is fine!